plugins/email_service/config/policy.json

```diff
@@ -1,3 +1,30 @@
 {
-  "email_service:application_list": "@"
+  "email_service_user": "role:email_user",
+  "email_service_admin": "role:email_admin",
+  "email_role_required": "role:email_user or role:email_admin",
```

Comment:
As far I can see there is no difference for any of the actions described here between email service user oder email admin? Is this right? Why do you have then two roles?

Reply:
Disabling and enabling the cronus require `email_admin` role. All other functionalities merely require `email_user` role.